client/components/CreateGroup.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+
+/**
+ * @class CreateGroup
+ * @author Babatunde Adeyemi <tundewrites@gmail.com>
+ */
+class CreateGroup extends Component {
+
+  /**
+   * Initializes the initial component state
+   */
+  constructor() {
+    super();
+
+    this.initialState = {
+      groupName: '',
+      description: '',
+      banner: ''
+    };
+
+    this.state = this.initialState;
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  /**
+   * Handles onChange event on input fields
+   * @param {object} event
+   * @returns {void}
+   */
+  handleChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  /**
+   * Handles onSubmit event on form submission
+   * @param {object} form
+   * @returns {void}
+   */
+  handleSubmit(form) {
+    form.preventDefault();
+    this.setState(this.initialState);
+    $('#create-group').modal('close');
```

Comment:
I'm not sure this needs to be here, you should abstract the logic of opening and closing a modal to your `index.html`, just ensure you have the same `id` for all your modal